src/client.ts

```diff
@@ -671,8 +670,7 @@ export class Client extends EventEmitter {
         method: 'post',
         host: this.config.apiHost,
         authToken: this.config.authToken,
-        logger: this.logger,
-        customHeaders: this.config.customHeaders
```

Comment:
Removed these because they get added in regardless in `requestApi`, so it seemed a bit redundant for the same `customHeaders`